src/app/zap-templates/zcl/data-model/chip/onoff-cluster.xml

```diff
@@ -79,8 +79,8 @@ limitations under the License.
 
     <command source="client" code="0x40" name="OffWithEffect" optional="true">
       <description>The OffWithEffect command allows devices to be turned off using enhanced ways of fading.</description>
-      <arg name="EffectIdentifier" type="OnOffEffectIdentifier"/>
```

Comment:
This one I would not touch.  Once the spec cleanups get to this cluster, this will in fact be an enum type....

Reply:
Done.